h2o-algos/src/main/java/hex/schemas/GLMModelV3.java

```diff
@@ -225,6 +242,9 @@ public GLMModelOutputV3 fillFromImpl(GLMModel.GLMOutput impl) {
         }
       }
       coefficients_table.fillFromImpl(tdt);
+      if (impl.ubeta() != null)
```

Comment:
Are we testing `if (impl.ubeta() != null)` twice ? Here and on line `200` ? It could all be done in one `if` block.